kite-data/kite-data-spark/pom.xml

```diff
@@ -318,5 +318,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
```

Comment:
The Flume and HBase modules are specifically not built in our fork, so there was no need to modify their POMs. We don't use Spark, but there was a low cost to keeping it in there.